developer_manual/app_development/bootstrap.rst

```diff
@@ -138,6 +144,6 @@ class and query an instance like
     declare(strict_types=1);
 
     // Register the composer autoloader for packages shipped by this app, if applicable
-    @include_once __DIR__ . '/../vendor/autoload.php';
+    include_once __DIR__ . '/../vendor/autoload.php';
```

Comment:
@ChristophWurst Was there a reason to hide errors here?

Reply:
I don't think so